torch/onnx/experimental/fonnx/v1/export_options.py

```diff
@@ -0,0 +1,34 @@
+import dataclasses
+from typing import Optional, Union
+
+
+@dataclasses.dataclass
+class ExportOptions:
+    """Class for keeping track of ONNX exporter flags can perform."""
+
+    opset_version: Optional[int] = None
+    """opset_version: Target ONNX opset version
+
+    When not specified, the exporter selects the latest supported version."""
+
+    log_level: Optional[Union[str, int]] = None
```

Comment:
I did not realize this wasn't an enum. I'm on the fence now about introducing our own enum because of that. Thoughts on `Union[ExportLogLevel, str, int]` and we map the python `str` and `int` variants to `ExportLogLevel` which is actually stored on the class? Not sure how to do this with dataclasses, but:

```python

class ExportLogLevel(enum):
    ...

class ExportOptions:
    _log_level: ExportLogLevel

    @property
    def log_level(self) -> ExportLogLevel:
        return self._log_level

    @log_level.setter
    def log_level(self, log_level: Optional[Union[ExportLogLevel, str, int]]):
        # do the mapping to self._log_level, raise on invalid values
        ...

Reply:
lgtm